### PR TITLE
Address build failures

### PR DESCRIPTION
**Pull request by MaxKsyunz**
Xcontent classes were moved in this PR https://github.com/opensearch-project/OpenSearch/pull/5902 causing builds to fail.

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by MaxKsyunz** [comment]
`SQL Java CI` still fails because `ml-commons` needs to be updated first. See https://github.com/opensearch-project/ml-commons/pull/741

**Review by acarbonetto — DISMISSED** [review]
LGTM! Thanks Max

**Review by rupal-bq — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]
I propose to wait for `ml-commons`, then restart CI, then merge.

**Review by YANG-DB — APPROVED** [review]

**Review by matthewryanwells — APPROVED** [review]

**Review by Yury-Fridlyand — DISMISSED** [review]

**Review by acarbonetto — APPROVED** [review]

**Review by matthewryanwells — DISMISSED** [review]

**Comment by Yury-Fridlyand** [comment]
```
2023-03-02T23:07:52.3208450Z [ant:checkstyle] [ERROR] /Users/runner/work/sql/sql/plugin/src/main/java/org/opensearch/sql/plugin/rest/RestQuerySettingsAction.java:25:1: Wrong lexicographical order for 'org.opensearch.common.xcontent.XContentType' import. Should be before 'org.opensearch.core.xcontent.XContentParser'. [CustomImportOrder]
2023-03-02T23:07:52.3209720Z 
2023-03-02T23:07:52.3219040Z > Task :opensearch-sql-plugin:checkstyleMain FAILED
```

File: opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/dsl/BucketAggregationBuilderTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.opensearch.common.bytes.BytesReference;
-import org.opensearch.common.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentBuilder;
```
Comment (Yury-Fridlyand) [inline opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/dsl/BucketAggregationBuilderTest.java:38]: You need to reorder imports

**Review by penghuo — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Comment by Yury-Fridlyand** [comment]
I think no backport to 2.x is needed.

**Pull request merged**